html/partials/compare/repositories.html.haml

```diff
@@ -0,0 +1,51 @@
+- if @diff["repositories"]
+  %a.scope_anchor{ id: "repositories" }
+  #repositories_container.scope
```

Comment:
Classes should be listed before IDs (.scope should precede #repositories_container)